src/bot/commands/report.ts

```diff
@@ -239,8 +161,8 @@ export default class ReportCommand extends Command {
               type: ComponentType.SelectMenu,
               customId: 'duplicate',
               placeholder: 'Choisissez un doublon...',
-              options: ratings.map((value) => ({
-                label: `Ressemblance à ${(value.rating * 100).toFixed(0)} %`,
+              options: ratings.map((value, index) => ({
+                label: `${index}. Ressemblance à ${(value.rating * 100).toFixed(0)} %`,
```

Comment:
L'index commence par 0
```suggestion
                label: `${index + 1}. Ressemblance à ${(value.rating * 100).toFixed(0)} %`,
```

Reply:
Ah oui c'est vrai... Je commit directement